tools/test.sh

```diff
@@ -11,6 +11,15 @@ fi
 cmd=$1
 shift;
 
+# Pass params to prepare, but omit potential --timeout
+# parameters as babel does not accept these.
+additionalParamsWithoutTimeout=( )
+for arg in $@; do
+  shift
+  [ "${arg:0:9}" = "--timeout" ] && continue
+  additionalParamsWithoutTimeout+=("$arg")
+done
```

Comment:
Depending on the result of our discussion in https://github.com/scalableminds/webknossos/pull/5730 this might can be removed